src/components/Editor/TextEditor.tsx

```diff
@@ -42,52 +33,72 @@ const TextEditor = (): JSX.Element => {
       setCurrDesc("");
     }
   }, [selectedAction, actionList]);
-  return isExpanded ? (
-    <TextDiv>
-      <Header>
-        <IconButton
-          sx={{ color: "white" }}
-          onClick={() => {
-            setExpanded((value) => !value);
-          }}
-        >
-          <ExpandIcon sx={{ fontSize: 30, ml: "0.5em" }} />
-        </IconButton>
-        <IconButton
-          sx={{ color: "white" }}
-          onClick={() => setDescription(currDesc)}
-        >
-          <SaveAltIcon sx={{ fontSize: 30 }} />
-        </IconButton>
-      </Header>
-      <Box
-        sx={{
-          m: "1em",
-          mt: "0.5em",
-          mb: "1.2em",
-          mr: "1.3em",
-          alignItems: "center",
-          height: "100%",
-        }}
-      >
-        <textarea
-          style={{
-            background: "white",
-            borderRadius: "5px",
-            width: "100%",
+  return (
+    <Rnd
+      cancel=".noDrag"
+      minWidth="200"
+      minHeight="150"
+      maxWidth="400"
+      maxHeight="250"
+    >
+      {" "}
```

Comment:
I'm not sure if this is where it causes the "problem". When I try to change the size of the text box, by doing a simple click on the boarder of the box, it will automatically change the size, growing in height. I think it's due to the minHeight not being in sync with the textBoxGrid height grid span. It's not a big deal. But a little surprising.

Reply:
the problem is with the positioning of the text editor in the parent dive. I fixed that in a new PR